front/cypress/e2e/idea_edit_page.cy.ts

```diff
@@ -109,6 +109,7 @@ describe('Idea edit page', () => {
 
     // save the form
     cy.get('.e2e-submit-idea-form').click();
+    cy.get('#e2e-accept-disclaimer').click();
```

Comment:
Do you think we need a test somewhere for when the disclaimer isn't being shown? Maybe we already have one implicitly though :thinking:

Reply:
Good question! It's tested for the proposals already but not for ideas. Doesn't feel critical enough of a feature to create a whole separate e2e test for the absence of a disclaimer but happy to add one if you disagree.

Reply:
No I think that's fine! :) :+1: